pkg/fanal/analyzer/command/apk/apk_test.go

```diff
@@ -28,284 +726,374 @@ func TestAnalyze(t *testing.T) {
+				config: &v1.ConfigFile{
```

Comment:
Perhaps move that config file to a [variable](https://github.com/aquasecurity/trivy/blob/2d3aedeecf539f221e3d73048dedb1d9071bec9e/pkg/fanal/analyzer/command/apk/apk_test.go#L22) (next to `alpineConifg`).
Then all test data will be in 1 place?

Reply:
Done
https://github.com/aquasecurity/trivy/pull/3434/commits/3e9e9aca3415853810bd867701c6e6081e6c0407